src/current/releases/index.md

```diff
@@ -60,6 +60,8 @@ The following binaries are not suitable for production environments:
 
         {% assign releases = site.data.releases | where_exp: "releases", "releases.major_version == v.major_version" | where_exp: "releases", "releases.release_type == s" | sort: "release_date" | reverse %} {% comment %} Fetch all releases for that major version based on release type (Production/Testing). {% endcomment %}
 
+{% comment %}Do a separate loop through the releases and for each release, copy some fields into some local v_ variables to use when we are not in a loop below{% endcomment %}
```

Comment:
does Liquid get confused if there is not a space between the closing `}` of comment and the first 'D' in "Do a ..."

similar question re: endcomment

Reply:
Nope :)